From ee401ba4d0d81fc244288e7e59a7bc5cfe2d1bf4 Mon Sep 17 00:00:00 2001
From: Quirin Gylstorff <quirin.gylstorff@siemens.com>
Date: Fri, 16 Jun 2023 20:32:38 +0200
Subject: [PATCH 4/4] d/rules: Add option to enable suricatta_wfx

Add the DEB_BUILD_PROFILE "pkg.swupdate.suricattawfx" to build with
suricattawfx lua module.

Signed-off-by: Quirin Gylstorff <quirin.gylstorff@siemens.com>
---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index 566a21bd..fc122d0d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -61,6 +61,9 @@ endif
 ifneq (,$(filter pkg.swupdate.suricattalua,$(DEB_BUILD_PROFILES)))
 	echo CONFIG_SURICATTA_LUA=y >> configs/debian_defconfig
 endif
+ifneq (,$(filter pkg.swupdate.suricattawfx,$(DEB_BUILD_PROFILES)))
+	echo CONFIG_SURICATTA_WFX=y >> configs/debian_defconfig
+endif
 ifneq (,$(LUA_VERSION))
 	echo CONFIG_LUAPKG=\"lua$(LUA_VERSION)\" >> configs/debian_defconfig
 	echo CONFIG_LUASCRIPTHANDLER=y           >> configs/debian_defconfig
-- 
2.39.2

